cyrw/bus_pico_pio.go

```diff
@@ -0,0 +1,79 @@
+//go:build pico && !cy43nopio
+
+package cyrw
+
+import (
+	"encoding/binary"
+	"machine"
+
+	pio "github.com/tinygo-org/pio/rp2-pio"
+	"github.com/tinygo-org/pio/rp2-pio/piolib"
+)
+
+var _busOrder = binary.LittleEndian
+
+type spibus struct {
+	cs  OutputPin
+	spi piolib.SPI3w
+}
+
+func DefaultNew() *Device {
+	// Raspberry Pi Pico W pin definitions for the CY43439.
+	const (
+		// WL_REG_ON = machine.GPIO23
+		// DATA_OUT  = machine.GPIO24
+		// DATA_IN   = machine.GPIO24
+		// IRQ       = machine.GPIO24 // AKA WL_HOST_WAKE
+		// CLK       = machine.GPIO29
+		// CS        = machine.GPIO25
+		WL_REG_ON = machine.GPIO23
+		DATA_OUT  = machine.GPIO24
+		DATA_IN   = DATA_OUT
+		CLK       = machine.GPIO29
+		CS        = machine.GPIO25
+	)
+	WL_REG_ON.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	CS.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	CS.High()
+	sm, err := pio.PIO0.ClaimStateMachine()
+	if err != nil {
+		panic(err.Error())
+	}
+	spi, err := piolib.NewSPI3w(sm, DATA_IN, CLK, 500_000)
+	if err != nil {
+		panic(err.Error())
+	}
+	spi.EnableStatus(true)
+	err = spi.EnableDMA(true)
```

Comment:
Are we doing DMA?

Reply:
Yes! DMA is active and seems to work

Reply:
How do we know it works?

Reply:
IT WORKS :smiley:  just figured it out :)